urwid/event_loop/abstract_loop.py

```diff
@@ -21,8 +19,11 @@
 #
 # Urwid web site: https://urwid.org/
 
+"""Abstract shared code for urwid IOLoop implementation."""
```

Comment:
IOLoop is a tornado thing, right? I think all the docs call these event loops http://urwid.org/manual/mainloop.html#event-loops

Reply:
Yep, copy-paste error. Fixed